PHP-Variables.md

```diff
@@ -0,0 +1,173 @@
+# PHP : Variables
+
+Variables are containers for storing data such as ```strings```, ```integers```, ```boolean``` values, ```array``` and objects.
+
+PHP follows certain rules for variable declarations such as:
+
+ * The variable must begin with a dollar sign ($)
+ 
+      Example: 
+
+  ```php
+<?php
+    $var = 5;
+?>
+```
+
+ * The variable name can contain characters such as A-Z, a-z, 0-9, _ and [ASCII](http://www.asciitable.com/ "ASCII Table") characters from 127-255.
+ 
+      Example:
+
+  ```php
+<?php
+   $var = 5; //Valid
+   $var_1 = "Foo"; //Valid
+   $_var2 = 'Bar'; //Valid
+    $var.3 = 'Baz'; //Invalid
+?>
+```
+
+ * The variable name can begin with underscore (_).
+  
+  Example:
+
+  ```php
+<?php
+     $_var2 = 'Bar'; //Valid
+?>
+```
+
+ * The variable name must not begin with a number 0-9.
+ 
+    Example:
+
+  ```php
+<?php
+     $9var3 = 'Baz'; //Invalid
+?>
+```
+
+* The variable name is case sensitive.
+  
+  Example:
+  
+```php
+<?php
+    $var = 5; //Valid
+    $VAR = "Foo"; //Valid
+    echo $var; //Output 5
+    echo "<br>";
+    echo $VAR; //Output Foo
+?>
+```
+
+ PHP is a loosely typed language, hence we don't need to declare the data type of a variable when declaring the variable. Unlike Java or C.
+
+```php
+<?php
+    $var = 5; 
+    $var2 = 4; 
+    $sum = $var+$var2;
+    echo $sum; //Output 9
+    echo "<br>";
+    echo $var+$var2; //Output 9
+?>
+```
+
+The variables can also be assigned by referencing.
+
+ Example :
+
+```php
+<?php
+    $var = 'Cat'; 
+    $var2 = &$var; 
+    $var2 = "My $var2";
+    echo $var2; //Output My Cat
+?>
```

Comment:
This could use a better example.  I think that you want to show that modifying `$var2` modifies `$var`.

Also, you introduce the `&` operator without much explanation.